src/Microsoft.Dnx.Runtime/NuGet/Utility/PathUtility.cs

```diff
@@ -72,7 +72,75 @@ public static string GetRelativePath(string path1, string path2)
         /// </summary>
         public static string GetRelativePath(string path1, string path2, char separator)
         {
-            return UriUtility.GetPath(GetRelativeUri(path1, path2), separator);
+            if (string.IsNullOrEmpty(path1))
+            {
+                throw new ArgumentException(nameof(path1));
```

Comment:
The string you passed is the error message.... The second parameter is the argument name.